includes/core-changes/corefx/2.1/instantiate-struct.md

```diff
@@ -1,10 +1,10 @@
 ### Private fields added to built-in struct types
 
-Private fields were added to built-in struct types in [reference assemblies](../../../../docs/standard/assembly/reference-assemblies.md). As a result, in C#, struct types must always be instantiated by using the [new operator](../../../../docs/csharp/language-reference/operators/new-operator.md) or [default literal](../../../../docs/csharp/language-reference/operators/default.md#default-literal), or by initializing each of the private fields.
```

Comment:
As a user of a struct, I have no means to initialize its private fields, except calling a constructor.